docs/scripts/configuration-table.py

```diff
@@ -0,0 +1,172 @@
+import json
+import re
+from pathlib import Path
+
+PROJECT_ROOT = Path(__file__).parents[2]
+
+
+def _table_row(items, delimiter="|"):
+    return f"| {' | '.join(items)} |"
+
+
+def _table_header():
+    return [
+        _table_row(
+            (
+                "Choice",
+                "Sub-field",
+                "Description",
+                "More information",
+            )
+        ),
+        _table_row(["---"] * 4),
+    ]
+
+
+def _new_section(item, item_type, default, description, more_info=""):
+    return [
+        f"## {item.replace('_', ' ').title()}",
+        "",
+        f"**Type:** {item_type}",
```

Comment:
My only potential build here is to also include the literal name of the `item` as a `Prompt: {item}` field, but that may be redundant / unnecessary.